walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,78 @@
 
 package com.google.sps;
 
+import java.util.*; 
 import java.util.Collection;
+import com.google.sps.TimeRange;
+import com.google.common.collect.Iterables;
 
+/** 
+ * Determines a list of available TimeRanges to schedule a meeting
+ * given a MeetingRequest and list of events taking place that day. 
+ */
 public final class FindMeetingQuery {
+  
+  /** 
+   * Returns the open windows for which {@code request} can be scheduled in.
```

Comment:
This comment doesn't talk about what 'events' is needed for. Can it be expanded?

Reply:
fixed